src/ts/components/subtitleoverlay.ts

```diff
@@ -36,13 +46,18 @@ export class SubtitleOverlay extends Container<ContainerConfig> {
     player.addEventHandler(player.EVENT.ON_CUE_ENTER, (event: SubtitleCueEvent) => {
       let labelToAdd = subtitleManager.cueEnter(event);
 
-      if (this.previewSubtitleActive) {
-        this.removeComponent(this.previewSubtitle);
+      if (this.isCEA608) {
```

Comment:
Don't you think showing the cue with the wrong styling and then, styling and moving it can be an issue?